azure-pipelines-ui-tests.dfe.yml

```diff
@@ -0,0 +1,220 @@
+schedules:
+- cron: 0 5 * * 1,2,3,4,5
+  branches:
+    include:
+    - dev
+  always: true
+- cron: 0 7 * * 1,2,3,4,5
+  branches:
+    include:
+    - dev
+  always: true
+- cron: 0 19 * * 1,2,3,4,5
```

Comment:
I think all these cron strings can be merged? So you have one cron string like `0 5,7,19 * * 1,2,3,4,5` rather than repeating this three times?